sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Data-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Please only use present tense (this could -&gt; this can). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
done